src/embed/dispatch.rs

```diff
@@ -24,24 +20,18 @@ pub struct Request {
     pub lc: LogContext,
     db_name: String,
     pub rpc: String,
-    pub data: String,
+    pub data: JsValue,
     pub response: Sender<Response>,
 }
 
+unsafe impl Send for Request {}
+
 type Response = Result<String, String>;
 
 lazy_static! {
     static ref SENDER: Mutex<Sender::<Request>> = {
         let (tx, rx) = channel::<Request>(1);
-
-        #[cfg(target_arch = "wasm32")]
         spawn_local(dispatch_loop(rx));
-
-        #[cfg(not(target_arch = "wasm32"))]
```

Comment:
Are we sure this leaves non-wasm dispatch in a usable state? Per https://github.com/rocicorp/repc/issues/155 we are *not* running dispatch tests in not-wasm (everyone keeps shrugging this off but it seems important).